### PR TITLE
`tools` version update to `0.8.12-SNAPSHOT`

### PR DESCRIPTION
**Pull request by dmytro-grankin**
This PR advances the `tools` version to `0.8.12-SNAPSHOT`, that contains the [changes](https://github.com/SpineEventEngine/tools/issues/65).

### CODE REVIEW

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

File: build.gradle [hunk 1]
```diff
@@ -118,14 +118,14 @@ subprojects {
     apply plugin: 'com.google.protobuf'
     apply plugin: 'java'
 
-    apply plugin: 'org.spine3.tools.javadoc-checker'
+    apply plugin: 'org.spine3.tools.spine-codestyle-checker'
     // Javadoc checker plugin allows finding FQN links with the wrong format.
```
Comment (armiol) [inline build.gradle]: This description should be updated as well.

In particular, we need to explain why this text is about links, but the plugin name is `checkCodestyle`. Or rewrite the text.

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

**Pull request merged**